modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_1_3/DDMStorageLinkUpgradeProcess.java

```diff
@@ -52,6 +52,13 @@ protected void doUpgrade() throws Exception {
 					connection,
 					"update DDMStorageLink set structureVersionId = ? where " +
 						"storageLinkId = ?");
+			PreparedStatement preparedStatement5 = connection.prepareStatement(
+				StringBundler.concat(
+					"select distinct DDMStorageLink.structureId, (select ",
+					"max(structureVersionId) from DDMStructureVersion where ",
+					"DDMStructureVersion.structureId = ",
+					"DDMStorageLink.structureId) as structureVersionId from ",
+					"DDMStorageLink"));
```

Comment:
@vngoliveira Is this `Select` the same from the previous code before [this change](https://github.com/liferay-forms/liferay-portal/commit/bc3de249c29b4b40649ceae447865efe4be4dae1#diff-847f87055f8db9aabab04cfad81ef2cdc77bb565a7a12634d449b4dc3e9bc726L24-L32)?

Reply:
Yes, it is